### ISSUE TITLE
[V3-alpha.8] Status bar space on android 4.4

### ISSUE THREAD

**Issue by yfuks** [wix/react-native-navigation#5336]
### Issue Description

Space between top bar and status bar on android 4.4

### Steps to Reproduce / Code Snippets / Screenshots
<img width="356" alt="Screenshot 2019-07-30 at 16 27 39" src="https://user-images.githubusercontent.com/9282988/62137900-ff7b3e80-b2e6-11e9-8378-849c7b82c8a0.png">

(Top bar is green and status bar is black)

---
### Environment
* React Native Navigation version: 3.0.0-alpha.8
* React Native version: 0.60.4
* Platform(s) : Android
* Device info: Simulator Nexus 4 API 19 (4.4) Debug

**Comment by mtzfactory** [comment]
Hi,

It happens in the Android Emulator as well...

```
Name: Nexus_5X_API_28_x86
CPU/ABI: Google APIs Intel Atom (x86)
Target: google_apis [Google APIs] (API level 28)
avd.ini.displayname: Nexus 5X API 28 x86
```

**Comment by bneigher** [comment]
I'm also seeing this - also using 3.0.0-alpha-8.  Happens to me on Samsung Galaxy S4

**Comment by bneigher** [comment]
Seems to be happening on Android API <=19 as it's fine on most modern API versions

**Comment by guyca** [comment]
Hey guys,
I'm sorry this wasn't communicated properly - RNN officially supports API 21 and above.

IMO, supporting API 19/20 devices is not worth it for two reasons:
1. The user base is very small and doesn't justify the effort
2. RN doesn't perform well on these low end devices

That being said, I'm open to accepting a PR to fix this issue.

**Comment by bneigher** [comment]
@guyca ok interesting.  I'm happy to take a crack at it though I don't have any idea what that white space is even called.  Any tips would be appreciated.

BTW - I have a 2014 SG S4 on API 19 and it's performing amazingly with react-native-navigation.  The only issue I'm having is this status bar issue.

**Comment by Agungkes** [comment]
Hey any update on this?
I've tried this today on android 4.4 (API 19) and android 5.1.1 (API 22) according to https://developer.android.com/about/dashboards.

On Android 4.4 I get problems like @bneigher  and also on Android 5.1.1 I get the same problem.
I tried to use topMargin and it runs properly, but when I remove statusBar (set drawBehind: true and visible: false), the topBar will run improperly and there is a distance between topBar and the content

note. i have not tried it on higher API
